### DISCUSSION TITLE
Error output when “ save_initial = True ”

### DISCUSSION THREAD

**Discussion by suyx0229** [iiasa/CWatM#128] [Q&A] [answered]
Dear CWatM Community:
       When I was practicing CWatM with the YouTube video, when I changed "save_initial" to True, the following error occurred when running the model. Is there any effective solution?
![微信图片_20240522143805](https://github.com/iiasa/CWatM/assets/169023319/59a07de2-5c3e-44e2-9e97-0765c5330bf3)

**Comment by weixiaoxun** [comment]
I have the same problem as you! ! ! I haven't found a solution yet, ><

**Reply by suyx0229** [reply]
I tried in the dynamic() method, adding a print statement to confirm the length of initVar after each iteration and check if it is the same length as initCondVar and initCondVarValue. The result is normal, but the following error occurs (as shown in the picture), but the magical thing is that the nc file of the stepinit time will be added to the init folder, but it may be wrong, and the model will end at the time set by Stepinit. It will not run any further.
Hope the professor can see this comment and provide some solutions to help us！！！
![微信图片_20240522214815](https://github.com/iiasa/CWatM/assets/169023319/7801081c-11e6-42bc-9c86-d0f01a87d264)

**Reply by weixiaoxun** [reply]
Yes, I discovered that too. When I followed the simulation on YouTube before, it seemed to be no problem. Now that I've changed to my own area, I don't know what's going on. Hope the professor can see our problem!

**Reply by suyx0229** [reply]
Unfortunately, I also followed the Danube watershed simulation done on YouTube, but this problem still occurred.
Looking forward to the professor's reply

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, thank you for pointing this out! We have now updated the [develop branch](https://github.com/iiasa/CWatM/tree/develop) which should fix this.
Let us know how it goes 👍🏼

**Reply by suyx0229** [reply]
Thank you very much to the professor for seeing the problem and giving a reply. I updated your model file and the problem has been solved after re-running it.
Thanks for the reply Professor ❤️
Best wishes

**Reply by weixiaoxun** [reply]
Me too,Thank you for replying us!!!!
Best wishes

**Reply by mikhailsmilovic** [reply]
Of course 💙 Looking forward to what you both explore with CWatM!
